recipes/setup_chef_server.rb

```diff
@@ -43,9 +43,9 @@
 # converge and complete the install.
 machine chef_server_hostname do
   if node['delivery-cluster']['chef-server']['existing']
-    recipe 'chef-server-12::delivery_setup'
+    run_list node['delivery-cluster']['chef-server']['run_list_existing']
   else
-    recipe 'chef-server-12'
+    run_list node['delivery-cluster']['chef-server']['run_list_new']
```

Comment:
For shimming in Ohai plugins to make vagrant use the 192.168* addresses rather than the 10.0.2.5 address that is the default eth0 ip address.